test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -427,7 +427,16 @@ protected void randomIndexTemplate() {
             // otherwise, use it, it has assertions and so on that can find bugs.
             SuppressCodecs annotation = getClass().getAnnotation(SuppressCodecs.class);
             if (annotation != null && annotation.value().length == 1 && "*".equals(annotation.value()[0])) {
-                randomSettingsBuilder.put("index.codec", randomFrom(CodecService.DEFAULT_CODEC, CodecService.BEST_COMPRESSION_CODEC));
+                randomSettingsBuilder.put(
+                    "index.codec",
+                    randomFrom(
+                        CodecService.DEFAULT_CODEC,
+                        CodecService.BEST_COMPRESSION_CODEC,
+                        CodecService.LUCENE_DEFAULT_CODEC,
+                        CodecService.ZSTD_CODEC,
+                        CodecService.ZSTD_NO_DICT_CODEC
```

Comment:
create a static list in this base class and use it all the other integ test classes